tests/bears/BearTest.py

```diff
@@ -4,7 +4,10 @@
 import unittest
 from os.path import abspath, exists, isfile, join, getmtime
 import shutil
-from time import sleep
+
+import datetime
+from freezegun import freeze_time
+
```

Comment:
please remove this newline so imports are at least divided by a single newline ;) this eases distinction between imports and tests ;)